volatility3/framework/layers/intel.py

```diff
@@ -354,10 +354,8 @@ def _get_invalid_pte_mask(self, kernel):
     @functools.lru_cache()
     def _get_PageFileLow_shift(self, kernel):
         mmpte_software_type = kernel.get_type("_MMPTE_SOFTWARE")
-        if mmpte_software_type.vol.members.get("SwizzleBit", None) is None:
-            return 1 # The old shift
 
-        return 12 # The new shift
+        return mmpte_software_type.vol.members["PageFileLow"][1].start_bit
```

Comment:
This is a pretty nasty way of doing it, and tinkers with the internal state of volatility.  I've now added #777 which gives the `child_template` call, so you can do `mmpte_software_type.child_template('PageFileLow').start_bit` instead.  Should be much cleaner and make it easier for us if we ever need to mess with the members mechanism.

Reply:
When are you planning on merging this in?